app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,43 @@
 class PostsController < ApplicationController
   def index
-    @users = User.find(params[:user_id])
-    @posts = Post.where(author_id: @users.id).order(created_at: :desc)
+    @user = User.find_by(id: params[:user_id])
+    @posts = Post.all.where(author_id: params[:user_id])
   end
 
   def show
-    @users = User.find(params[:user_id])
-    @posts = Post.find(params[:id])
+    @user = User.find_by(id: params[:user_id])
```

Comment:
- [ ] Following the project requirements, a post should be created on behalf of the currently logged-in user (current_user) of the app. Please use the current_user method that you created in the application_controller to create a post rather than using params[:user_id].

Reply:
Thank you, I did that